docs/pim/create_edit_product.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Create new products or modify existing ones.
+---
+
+# Create and edit products
+
+[Products](products.md#products) are a specific kind of [Content items](../content_management/content_items.md#content-items) that you use 
+to present your offer in the website, including product specification, and pricing.
+Individual products are instances of [Product Types](create_product_types.md#create-product-types).
+You can only create or modify products when your [user role](../permission_management/work_with_permissions.md) has the `Product/Edit` permission.
+
+To create a product, depending on how the Product Type is defined, you [may need to provide](products.md#product-completeness) certain pieces 
+of information in their respective [Fields](../content_management/content_model.md#fields-and-field-types).
+
+You can create product by using manual or bulk method.
+Bulk method can be used only at the developer level. See [Products](https://doc.ibexa.co/en/latest/pim/product_api/#products) for a technical guide on how to do this.
+
+## Manually create product
+
+You can create product in several ways:
+
+- Click **Create** in the upper-right corner of the [Products](products.md) screen. 
+- While viewing a **Product Catalog** in the [Content Tree](../getting_started/discover_ui.md#content-tree), click **Create content** at the top of the screen.
+The new item appears in the tree.
+- Click **Create content** in the upper-right corner of the [My dashboard](../getting_started/discover_ui.md) screen. In this case you have to choose Content Type and select where the content will be located.
+
+1\. If you are adding a new product, use one of the methods mentioned above and skip to step 3.
+
+![Adding a Product](img/add_product.png "Adding a Product")
```

Comment:
```suggestion
```

I think I would remove this screenshot. Instead, I would add the following image in step 3, with caption "Creating a new product".
![obraz](https://user-images.githubusercontent.com/64841871/209154286-4348ebd6-a90e-49d8-ba2c-a1e96b45aea8.png)